src/components/Registration/Registration.jsx

```diff
@@ -0,0 +1,148 @@
+import React, { Component } from 'react';
```

Comment:
Since this class is doing api calls and gathering information, it would be more appropriate to be a container instead of a component. In general, components only get information from its props.

Reply:
By Container you do you meant just have another component that wraps it and provides the API call as a prop? Or is this a redux specific thing?

Reply:
You can either do another component that wraps it or just move this component into the containers folder since this itself can be a container.